config.json

```diff
@@ -70,6 +70,7 @@
             "env_aws_actions": true,
             "cdk_pivot_role_multiple_environments_same_account": false,
             "enable_quicksight_monitoring": false
-        }
+        },
+        "log_query_period_days": 1
```

Comment:
Maybe a better name for this would be logViewPeriodDays or logDisplayRangeDays since this variable is specifically for displaying logs going back "x" days.

Reply:
++ if this is only for shares I would move not under `core` - maybe `datasets_base` because we have share feature flags there already (and no `shares_base`)

Reply:
I would prefer to leave name as it is, since it's about query, not about view. Only the last one log is displayed, even if there were 10 of them.

@noah-paige as we discussed in slack, it's not only for shares, but for all logs, so we leave it in the core